ci/kmm-kmod-dockerfile.yaml

```diff
@@ -7,6 +7,8 @@ data:
     FROM ubuntu as builder
 
     ARG KERNEL_VERSION
+    ARG MOD_NAME
```

Comment:
what is the use of those parameters during build?

Reply:
We don't really have one in our CI example, but I think we should have all substitutions in the `containerImage` also available as default build arguments to avoid confusion.

Reply:
Correct, if we are not using in our CI dockerfile, we can remove them :+1: